hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -107,6 +107,7 @@
   // comma separated list of input files
   public static final String INPUT_FILES =
       "mapreduce.input.carboninputformat.files";
+  public static final String ALTER_PARTITION_ID = "mapreduce.input.carboninputformat.partitionid";
```

Comment:
I have migrate all the changes to CarbonTableInputFormat. Just keep the modification in CarbonInputFormat. It's already not used and you can remove it safely.
But sure, I can recover this file if it's necessary.